src/ompl/base/OptimizationObjective.h

```diff
@@ -33,6 +33,7 @@
 *********************************************************************/
 
 /* Author: Luis G. Torres, Ioan Sucan */
+/* Edited by: Jonathan Gammell (allocInformedStateSampler) */
```

Comment:
I think you deserve full authorship status on all the files you've contributed to, but I'll defer to Ioan and Mark on this.